iaso/api/completeness.py

```diff
@@ -19,12 +21,18 @@ class CompletenessViewSet(viewsets.ViewSet):
 
     def list(self, request):
         profile = request.user.iaso_profile
+        profile_org_units = OrgUnit.objects.filter(id__in=profile.org_units.all())
```

Comment:
Is there a reason for not using `OrgUnit.objects.filter_for_user(request.user)` instead?

Reply:
This proposition **OrgUnit.objects.filter_for_user(request.user)** will return all the org units in the pyramid(tree) and if I apply the filter on all the org units like this **queryset = queryset.filter(org_unit__in=profile_org_units))**, from my point of view it will slow down a litter bit the loading of the page.

Reply:
Ok, I thought it was more readable. Sometimes, sacrificing a little performance for readability can be worth it!

But I won't block the PR for this :)

Reply:
@kemar ,for the readability, let's sacrifice the little performance. thank you so much